enterprise/server/remote_execution/operation/operation.go

```diff
@@ -110,6 +111,12 @@ func ExecuteResponseWithResult(ar *repb.ActionResult, err error) *repb.ExecuteRe
 	if ar != nil {
 		rsp.Result = ar
 	}
+	if target, err := flagutil.GetDereferencedValue[string]("executor.app_target"); err == nil && target == "grpcs://cloud.buildbuddy.io" {
+		if rsp.GetMessage() != "" {
+			rsp.Message = rsp.GetMessage() + "\n"
+		}
+		rsp.Message = rsp.GetMessage() + "This executor is using the old BuildBuddy endpoint, cloud.buildbuddy.io. Migrate `executor.app_target` to remote.buildbuddy.io for improved performance."
+	}
```

Comment:
in practice, when does this message get displayed? does it show up once per action that is executed, or only once per invocation? (or only when there are other errors with the invocation?)

Reply:
I think this shows up once per action, which is probably too much. I'll change it to show up just at the end.